var/spack/repos/builtin/packages/flatbuffers/package.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Flatbuffers(CMakePackage):
+    """Memory Efficient Serialization Library
+    """
+
+    homepage = "http://google.github.io/flatbuffers/"
+    url      = "https://github.com/google/flatbuffers/archive/v1.9.0.tar.gz"
+
+    version('1.9.0', '8be7513bf960034f6873326d09521a4b')
+    version('1.8.0', '276cab8303c4189cbe3b8a70e0515d65')
+
+    variant('build_type', default='Release')
+
+    def cmake_args(self):
+        args = []
+        return args
```

Comment:
if you don't need any arguments, you can delete this method. No need to implement.